net.ptilouk.superfluous.yml

```diff
@@ -0,0 +1,117 @@
+app-id: net.ptilouk.superfluous
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: superfluous-returnz
+rename-icon: superfluous-returnz
+rename-desktop-file: superfluous-returnz.desktop
+rename-appdata-file: superfluous-returnz.metainfo.xml
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=all
+  - --share=network
```

Comment:
why does it need network?

Reply:
It doesn't actually (I thought it did because there's a link to the website inside the game, but this obviously calls a browser and so the game does not directly need network access), I'll remove it.